source/git/gitsource.go

```diff
@@ -671,6 +671,7 @@ func git(ctx context.Context, dir, sshAuthSock, knownHosts string, args ...strin
 			//	"GIT_TRACE=1",
 			"GIT_CONFIG_NOSYSTEM=1", // Disable reading from system gitconfig.
 			"HOME=/dev/null",        // Disable reading from user gitconfig.
+			"LC_ALL=C",              // Ensure consistent output.
```

Comment:
Feel free to ignore; not sure if it's redundant, and already clear from the `LC_ALL` (which most people would understand does).

```suggestion
			"LC_ALL=C",              // Ensure consistent, non-localized output.
```